config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: IMAGE_URL
+      - image: maestrosdk/controller:latest
```

Comment:
Is there a nice way to avoid latest and have more specific tags? I am just seeing a scenario where in a longrunning cluster :latest gets updated...

Reply:
The way to work with a specific tag via the make file is the TAG argument I added.  So running `TAG=0.2.0 make docker-build` will build the 0.2.0 tag.  Similar with other make calls e.g. deploy